lib/companies/jobs.ex

```diff
@@ -102,4 +102,20 @@ defmodule Companies.Jobs do
   def change(%Job{} = job) do
     Job.changeset(job, %{})
   end
+
+  def search(filters) do
+    filters
+    |> Enum.reduce(Job, fn
+      {_, nil}, query ->
```

Comment:
Let's make this a named function instead. I've never been a big fan of anonymous functions w/ many heads.

How about something like this? I _think_ we can also reduce this to just 2 matches:

```elixir
defp query_predicates({"text", text}, query), 
  do: from c in query, where: ilike(c.title, ^"%#{text}%")
defp query_predicates(_, query), 
  do: query
```

Reply:
Sure, I will do the change. Still adding stuff btw.